sources/packages/backend/libs/services/src/queue/queue.module.ts

```diff
@@ -91,3 +102,36 @@ function getQueueModules(): BullModuleAsyncOptions[] {
     inject: [ConfigService, QueueService],
   }));
 }
+
+/**
+ * Builds the Bull Board module options to register the dashboard in a dynamic way.
+ * @param configService service with the configuration of the application.
+ * @returns Bull Board module options with the dashboard route,
+ * authentication middleware and the board options.
+ */
+async function bullBoardModuleFactory(configService: ConfigService) {
```

Comment:
Please specify the return type in the method.

```ts
async function bullBoardModuleFactory(
  configService: ConfigService,
): Promise<BullBoardModuleOptions>
```

Reply:
Return type added.